apps/fpc/fsw/unit_test/fpc_test_utils.hpp

```diff
@@ -41,12 +41,13 @@
 #include "fpc_app.h"
 
 
-#if 1
-    #define     FPC_UT_TEST_WITH_OWN_FPC_OBJECT
-#else
-    #define     FPC_UT_TEST_WITH_EXTERN_APPMAIN
+#if 0
```

Comment:
Delete all the macros that have no effect.

Reply:
Yes, sir

Reply:
Done